src/developers/notary_server.md

```diff
@@ -67,14 +67,13 @@ We host multiple versions of the notary server:
 
 | Version        | Notary URL                            | Info/Status                                                                                                    | GitHub                                                                                       | Note                                                                           |
 | -------------- | ------------------------------------- | -------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------ |
+| v0.1.0-alpha.8 | https://notary.pse.dev/v0.1.0-alpha.8 | [info](https://notary.pse.dev/v0.1.0-alpha.8/info)/[health](https://notary.pse.dev/v0.1.0-alpha.8/healthcheck) | [v0.1.0-alpha.8](https://github.com/tlsnotary/tlsn/tree/v0.1.0-alpha.8/crates/notary/server) | [Release notes](https://github.com/tlsnotary/tlsn/releases/tag/v0.1.0-alpha.8) |
 | v0.1.0-alpha.7 | https://notary.pse.dev/v0.1.0-alpha.7 | [info](https://notary.pse.dev/v0.1.0-alpha.7/info)/[health](https://notary.pse.dev/v0.1.0-alpha.7/healthcheck) | [v0.1.0-alpha.7](https://github.com/tlsnotary/tlsn/tree/v0.1.0-alpha.7/crates/notary/server) | [Release notes](https://github.com/tlsnotary/tlsn/releases/tag/v0.1.0-alpha.7) |
 | nightly        | https://notary.pse.dev/nightly        | [info](https://notary.pse.dev/nightly/info)/[health](https://notary.pse.dev/nightly/healthcheck)               | [dev](https://github.com/tlsnotary/tlsn/tree/dev/crates/notary/server)                       |                                                                                |
 
 All the servers above run the TLSNotary notary software in a Trusted Execution Environment (TEE), Intel SGX on Azure. 
 You can verify the software attestation by visiting `https://notary.pse.dev/<version>/info` (for versions alpha.8-pre and later).
 
-For more details on the deployment, refer to this [GitHub Action](https://github.com/tlsnotary/tlsn/blob/main/.github/workflows/cd-server.yml).
```

Comment:
@heeckhau removing this since it's deleted, and the new tee cd is not ready yet I believe?

Reply:
correct.

I'll also have to fix that only alpha.8 and later run in TEEs. Not alpha.7